### DISCUSSION TITLE
Does Fortio support distributed testing  like JMeter?

### DISCUSSION THREAD

**Discussion by bernersyang** [fortio/fortio#733] [Q&A] [answered]
I wanted to confirm how to use multiple systems to perform stress testing by Fortio like JMeter，but no useful reference materials were found.

**Comment by ldemailly** [comment, marked as answer]
You can setup multiple fortio pods in kubernetes for instance, or in general on multiple machines and use the rest api to trigger load generation from each.

Note that this is rarely necessary because fortio is very efficient and a single fortio can easily overload most target systems.

Having a controller and aggregator is an open issue/improvement but hasn't been a priority because of the above.
